docs/sources/shared/datasources/sql-query-builder-macros.md

```diff
@@ -0,0 +1,19 @@
+---
+headless: true
+labels:
+  products:
+    - enterprise
+    - oss
+---
+
+#### Macros
+
+You can enable macros support in the select clause. This allows you to create timeseries queries.
+
+{{< docs/experimental product="Macros support in visual query builder" featureFlag="`sqlQuerybuilderFunctionParameters`" >}}
+
+Use the **Data operations** dropdown to select a macro like `$__timeGroup` or `$__timeGroupAlias`. Select a time column from the **Column** dropdown and a time interval from the **Interval** dropdown to create a time series query.
+
+{{< figure src="/media/docs/grafana/data-sources/screenshot-sql-builder-time-series-query.png" class="docs-image--no-shadow" caption="SQL query builder time series query" >}}
+
+You can also add custom value to the **Data operations**. For example a function that is not in the dropdown list. This will allow you to add any number of parameters.
```

Comment:
- I think these are all adjectival forms of "time series" so we should use "time-series": https://grafana.com/docs/writers-toolkit/write/style-guide/word-list/#time-series
- It's also "drop-down" rather than "dropdown": https://grafana.com/docs/writers-toolkit/write/style-guide/word-list/#drop-down
- I've also kept in the present tense (avoiding "will:) and made some minor edits for readability

```suggestion
You can enable macros support in the select clause to create time-series queries.

{{< docs/experimental product="Macros support in visual query builder" featureFlag="`sqlQuerybuilderFunctionParameters`" >}}

Use the **Data operations** drop-down to select a macro like `$__timeGroup` or `$__timeGroupAlias`.
Select a time column from the **Column** drop-down and a time interval from the **Interval** drop-down to create a time-series query.

{{< figure src="/media/docs/grafana/data-sources/screenshot-sql-builder-time-series-query.png" class="docs-image--no-shadow" caption="SQL query builder time-series query" >}}

You can also add custom value to the **Data operations**.
For example, a function that's not in the drop-down list.
This allows you to add any number of parameters.
```